packages/remote/lib/adapter.js

```diff
@@ -73,17 +73,33 @@ class FuryRemoteAdapter {
   }
 
   parse({
-    source, generateSourceMap, namespace, mediaType,
+    source, generateSourceMap, generateMessageBody, generateMessageBodySchema, namespace, mediaType,
   }) {
     const inputMediaType = mediaType || detectMediaType(source, defaultInputMediaType);
 
+    if (generateMessageBody === undefined) {
+      this.generateMessageBody = true;
+    } else {
+      this.generateMessageBody = generateMessageBody;
+    }
+
+    if (generateMessageBodySchema === undefined) {
+      this.generateMessageBodySchema = true;
+    } else {
+      this.generateMessageBodySchema = generateMessageBodySchema;
+    }
+
     return axios({
       method: 'post',
       url: this.options.parseEndpoint,
       baseURL: this.options.url,
       data: source,
       params: {
         generateSourceMap,
+        adapterOptions: {
+          generateMessageBody: this.generateMessageBody,
+          generateMessageBodySchema: this.generateMessageBodySchema,
+        },
```

Comment:
The options should be together as they should be in the querystring like the following:`?generateSourceMap=true&generateMessageBody=true`

```suggestion
        generateMessageBody: this.generateMessageBody,
        generateMessageBodySchema: this.generateMessageBodySchema,
```